src/main/java/uk/gov/hmcts/reform/managecase/domain/OrganisationPolicy.java

```diff
@@ -16,5 +17,7 @@
     private String orgPolicyReference;
     @JsonProperty("OrgPolicyCaseAssignedRole")
     private String orgPolicyCaseAssignedRole;
+    @JsonProperty("PreviousOrganisations")
+    private List<PreviousOrganisation> previousOrganisations;
```

Comment:
Has this been already updated in the definition store?

Reply:
Yes, this has been done on definition store.